src/test/resources/discovery/specification-sample-1.html

```diff
@@ -11,17 +11,20 @@ <h1>Contents</h1>
     </ul>
 </section>
 <section about="https://example.org/specification1#spec1" typeof="spec:NormativeRequirement">
-    <h1>Requirement 1</h1>
+    <h1 property="schema:name">Requirement 1</h1>
```

Comment:
Not sure if the spec will have a short name/label for the requirement. There is currently nothing of that kind in the specs right now as far as I know.

Reply:
I wasn't sure if it was useful but I put it in because it is already done in the protocol spec:
```
<section id="resource-representations" inlist="" rel="schema:hasPart" resource="#resource-representations">
  <h3 property="schema:name">Resource Representations</h3>
  <div datatype="rdf:HTML" property="schema:description">
  </div>
</section>
```

Reply:
What's in the Protocol spec is for the section name/heading.